src/app/pages/swap/hooks/use-bitflow-swappable-assets.tsx

```diff
@@ -57,8 +57,8 @@ function useCreateSwapAsset(address: string) {
 
       const availableBalance = sip10Tokens.find(
         sip10Token =>
-          getPrincipalFromAssetString(sip10Token.info.contractId) === token.tokenContract
-      )?.balance.availableBalance;
+          getPrincipalFromAssetString(sip10Token.asset.contractId) === token.tokenContract
```

Comment:
Just double checking this change of `balance.availableBalance` to `crypto.availableBalance` is intended